packages/fastify-vite/mode/production.js

```diff
@@ -127,19 +127,22 @@ async function setup(config) {
   async function loadBundle(viteConfig, distOutDir, entryPath) {
     const parsedNamed = parse(entryPath).name
     const bundleFiles = [`${parsedNamed}.js`, `${parsedNamed}.mjs`]
+
+    const fixWin32Path = process.platform === 'win32'
+      ? (filePath) => new URL(fileUrl(filePath))
+      : (filePath) => filePath
```

Comment:
shouldn't this be extracted outside of this function?
We are re-creating the `fix` function on every invocation of loadBundle, but I don't think that the platform will change once is executed the first time.